projects/laji/src/app/shared/model/ArrayResult.ts

```diff
@@ -0,0 +1,4 @@
+
+export interface ArrayResult<T> {
+  results: T[];
```

Comment:
This api is a bit strange. Why not return a simple array from the api?

Reply:
So we can give JSON-LD context to the result, without needing each item to have the context: https://w3c.github.io/json-ld-bp/#top-level-object